src/schemas/json/sublime-syntax.json

```diff
@@ -0,0 +1,248 @@
+{
+  "$comment": "https://www.sublimetext.com/docs/syntax.html",
+  "$id": "https://json.schemastore.org/sublime-syntax.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "definitions": {
+    "meta_pattern": {
+      "type": "object",
+      "properties": {
+        "meta_scope": {
+          "description": "This assigns the given scope to all text within this context, including the patterns that push the context onto the stack and pop it off.",
+          "type": "string"
+        },
+        "meta_content_scope": {
+          "description": "As above, but does not apply to the text that triggers the context (e.g., in the above string example, the content scope would not get applied to the quote characters).",
```

Comment:
"As above" will likely be seen out of context, I'd suggest copying the relevant parts of the referenced description or reference it by name to say how it differs.